lib/ts/recipe/passwordless/types.ts

```diff
@@ -449,3 +461,26 @@ export type APIInterface = {
         exists: boolean;
     }>;
 };
+
+export type TypeEmailDeliveryTypeInput = {
+    type: "PASSWORDLESS_LOGIN_CODE";
+    user: {
+        email: string;
+    };
```

Comment:
Why is the email in the user object? Keep the old function's input signature.

Reply:
okay